maintainers/maintainer-list.nix

```diff
@@ -10604,6 +10604,12 @@
     githubId = 11824817;
     name = "Marti Serra";
   };
+  xworld21 = {
+    email = "xworld21@users.sf.net";
```

Comment:
The anonymous github mail would be better incase sourceforge changes anything and we can't reach you or a mail directly from your mail provider.

Reply:
Huh, actually, it looks like my old alias has already stopped working. Thanks for pointing this out, I'll change email.